common/misc.hh

```diff
@@ -145,9 +145,6 @@ string itoa(int i);
 string uitoa(unsigned int i);
 string bitFlip(const string &str);
 
-void dropPrivs(int uid, int gid);
-int makeGidNumeric(const string &group);
-int makeUidNumeric(const string &user);
```

Comment:
The source for this apparently removed these functions as part of https://github.com/PowerDNS/pdns/commit/f1d6a7ced5f84adf750234f09da635ef54a53624

But I didn't look to find whether these were ever used here.

Reply:
They're not used currently in weakforced, but I'm not sure what that has to do with spelling?

Reply:
I try to correct things based on other instances, since this was the only instance, it was hard to decide if the item was misspelled or otherwise.

I can drop this if that's desired.